packages/button/index.js

```diff
@@ -26,7 +26,7 @@ class WarpButton extends kebabCaseAttributes(LitElement) {
     href: { type: String, reflect: true },
     target: { type: String, reflect: true },
     rel: { type: String, reflect: true },
-    buttonClass: { type: String, reflect: true },
```

Comment:
We have copied this component from Fabric DS and there I see we used to add `this.buttonClass` to Button's classes ([see here](https://github.com/fabric-ds/elements/blob/next/packages/button/index.js#L69)). I see that we removed `this.buttonClass` from the list in [this commit](https://github.com/warp-ds/elements/commit/0aae5004b7bb994610dc8ffb6afa8b274d6a1682). I have a feeling it was done by mistake but correct me if I'm wrong @AnnaRybkina. 
I checked that passing `w-full max-w-full` or `w-full! max-w-full!` to that prop would not fix the full-width issue anyway (`max-w-max` always takes precedence), but maybe there are other scenarios where overriding/adding classes to button from the outside would be needed?

Reply:
Yea I deleted it by mistake but it is not even in the docs so I haven't thought about it. It doesn't work with unocss as unocss on bulid time doesn't know what it is provided in `buttonClass` and if consumers won't penetrate shadow dom with the styles for the classes then we won't get any styling for anything provided in `buttonClass`. Hopefully this will be fixed by what Web Wlatform team is working on with the constructed stylesheets.
But yes, I agree let's put it back as before. Good point.